Triangulation_2/doc/Triangulation_2/Triangulation_2.txt

```diff
@@ -1124,6 +1124,21 @@ through the edge.
 
 \cgalExample{Triangulation_2/polylines_triangulation.cpp}
 
+\subsection Triangulation_2ExampleBuildingFromSoup Example: Building a Triangulated Arrangement of Polylines from a Segment Soup
+
+The `Constrained_triangulation_plus_2` structure is initialized by a
+set of polylines. As users may only be able to provide a disconnected
+segment soup as input, a member function
+[split_subconstraint_graph_into_constraints()](@ref Constrained_triangulation_plus_2::split_subconstraint_graph_into_constraints())
```

Comment:
```suggestion
[`split_subconstraint_graph_into_constraints()`](@ref Constrained_triangulation_plus_2::split_subconstraint_graph_into_constraints())
```

Reply:
No, for some reason, this doesn't work (the backquotes just end up showing in the HTML output, which is not the desired behavior).

Reply:
\link \endlink?

Reply:
Yes! This works :)